{"kind":"symbol","hierarchy":{"paths":[["doc:\/\/com.staq.TrustlessSDK\/documentation\/TrustlessSDK","doc:\/\/com.staq.TrustlessSDK\/documentation\/TrustlessSDK\/TrustlessError"]]},"schemaVersion":{"minor":3,"major":0,"patch":0},"metadata":{"symbolKind":"case","title":"TrustlessError.unexpected","roleHeading":"Case","externalID":"s:12TrustlessSDK0A5ErrorO10unexpectedyA2CmF","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"unexpected","kind":"identifier"}],"role":"symbol","modules":[{"name":"TrustlessSDK"}]},"sections":[],"identifier":{"url":"doc:\/\/com.staq.TrustlessSDK\/documentation\/TrustlessSDK\/TrustlessError\/unexpected","interfaceLanguage":"swift"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/trustlesssdk\/trustlesserror\/unexpected"]}],"abstract":[{"text":"Indicates an unexpected error occurred, details of which may not be available.","type":"text"}],"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unexpected","kind":"identifier"}],"platforms":["iOS"],"languages":["swift"]}]}],"references":{"doc://com.staq.TrustlessSDK/documentation/TrustlessSDK/TrustlessError":{"url":"\/documentation\/trustlesssdk\/trustlesserror","type":"topic","role":"symbol","abstract":[{"type":"text","text":"Describes errors that can occur within the TrustlessSDK."}],"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"TrustlessError","kind":"identifier"}],"identifier":"doc:\/\/com.staq.TrustlessSDK\/documentation\/TrustlessSDK\/TrustlessError","navigatorTitle":[{"text":"TrustlessError","kind":"identifier"}],"title":"TrustlessError","kind":"symbol"},"doc://com.staq.TrustlessSDK/documentation/TrustlessSDK":{"role":"collection","kind":"symbol","title":"TrustlessSDK","type":"topic","url":"\/documentation\/trustlesssdk","identifier":"doc:\/\/com.staq.TrustlessSDK\/documentation\/TrustlessSDK","abstract":[]},"doc://com.staq.TrustlessSDK/documentation/TrustlessSDK/TrustlessError/unexpected":{"kind":"symbol","title":"TrustlessError.unexpected","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"unexpected","kind":"identifier"}],"role":"symbol","identifier":"doc:\/\/com.staq.TrustlessSDK\/documentation\/TrustlessSDK\/TrustlessError\/unexpected","type":"topic","url":"\/documentation\/trustlesssdk\/trustlesserror\/unexpected","abstract":[{"text":"Indicates an unexpected error occurred, details of which may not be available.","type":"text"}]}}}